packages/plugins/test/mocha/simulate.spec.ts

```diff
@@ -50,15 +62,37 @@ describe('Simulate', () => {
   // orders of magniture higher than standard local nodes, which caused a bug in the simulation
   // logic.
   it('succeeds on every live supported network', async () => {
-    const results = await Promise.allSettled(
-      parsedConfigArray.map((parsedConfig) =>
-        simulate(
-          parsedConfigArray,
-          parsedConfig.chainId,
-          fetchURLForNetwork(toSupportedChainId(Number(parsedConfig.chainId)))
+    // We run the simulation for five networks at a time. We can't batch all of the networks at once
+    // because of rate limits.
+    const batchSize = 5
+    const batches: Array<Array<ParsedConfig>> = []
+
+    // Create batches
+    for (let i = 0; i < parsedConfigArray.length; i += batchSize) {
+      const batch = parsedConfigArray.slice(i, i + batchSize)
+      batches.push(batch)
+    }
+
+    const results: Array<
+      PromiseSettledResult<{
+        receipts: SphinxTransactionReceipt[]
+        batches: SphinxLeafWithProof[][]
+      }>
+    > = []
+
+    // Process each batch sequentially
+    for (const batch of batches) {
+      const batchResults = await Promise.allSettled(
+        batch.map((parsedConfig) =>
+          simulate(
+            parsedConfigArray,
+            parsedConfig.chainId,
+            fetchURLForNetwork(toSupportedChainId(Number(parsedConfig.chainId)))
+          )
```

Comment:
We now batch the simulations because our CI process was rate-limited by Alchemy when we ran all of the simulations in parallel